plugins/postcss-logical/src/lib/transform-borders.ts

```diff
@@ -56,25 +57,44 @@ export function transformBorderShorthand(
 }
 
 export function transformBorderRadius(
-	start: string,
-	end: string,
+	config: DirectionConfig,
 ): (declaration: Declaration) => boolean {
 	return (declaration: Declaration) => {
 		let prop;
 
 		switch (declaration.prop.toLowerCase()) {
-			case 'border-start-start-radius':
-				prop = `border-top-${start}-radius`;
+			case 'border-start-start-radius': {
+				const replacement = config.inlineIsHorizontal ?
+					`${config.block[0]}-${config.inline[0]}` :
+					`${config.inline[0]}-${config.block[0]}`;
```

Comment:
- First `start` is always `block`.
- So we have to take the value at index `0` from `config.block`.
- Second `start` is always `inline`.
- So we have to take the value at index `0` from `config.inline`.

The result might be : `left-bottom`.
So we check if the orientation is different and inverse the parts if needed.